en/mozillavpn.xlf

```diff
@@ -0,0 +1,505 @@
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file datatype="plaintext" original="../src/components/VPNCheckBoxAlert.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.turnOffAlert.enabling">
+      <source xml:space="preserve">VPN must be off before enabling</source>
+      <context-group purpose="location"><context context-type="linenumber">32</context></context-group>
+      <note annotates="source" from="developer">This is a Setting which requires the VPN to be disconnected to chance state</note>
+    </trans-unit>
+    <trans-unit id="vpn.turnOffAlert.disabling">
+      <source xml:space="preserve">VPN must be off before disabling</source>
+      <context-group purpose="location"><context context-type="linenumber">35</context></context-group>
+      <note annotates="source" from="developer">This is a Setting which requires the VPN to be disconnected to chance state</note>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/components/VPNConnectionInfo.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="ip">
+      <source xml:space="preserve">IP: %1</source>
+      <context-group purpose="location"><context context-type="linenumber">114</context></context-group>
+      <note annotates="source" from="developer">The Current Ip adress</note>
+    </trans-unit>
+    <trans-unit id="download">
+      <source xml:space="preserve">download</source>
+      <context-group purpose="location"><context context-type="linenumber">126</context></context-group>
+      <note annotates="source" from="developer">the current download speed</note>
```

Comment:
These comments should provide a bit more context. For example, is it followed by the actual speed? Same line or new line?